README.md

```diff
@@ -61,8 +61,9 @@ An <a id="online-image" href="#online-image">online image</a> URL consists of ei
 
 which ends with:
 
-- a period, followed by one of the following path extensions: `png` `jpg` `jpeg` `gif` `tif` `tiff`.
-- optionally followed by a query string.
+- a period `.`, followed by one of the following case-insensitive path extensions: `png` `jpg` `jpeg` `gif` `tif` `tiff`;
+- optionally followed by a query string starting with `?`;
+- optionally followed by a fragment identifier starting with `#`, e.g. for ![Media Fragments](https://www.w3.org/TR/media-frags/).
```

Comment:
Please remove `, e.g. for ![Media Fragments](https://www.w3.org/TR/media-frags/)`.